logic/admin.go

```diff
@@ -0,0 +1,50 @@
+package logic
+
+import (
+	"fmt"
+
+	"github.com/zorchenhimer/MoviePolls/models"
+)
+
+// Purge removes the account entirely, including all of the account's votes.
+// Should this add the user to the banlist?  Maybe add an option?
```

Comment:
I like the option idea.  This could be a checkbox in the UI.

(UI stuff would come later)

Reply:
Well that comment is from you, i didnt add it :)

Reply:
LOL